addon/xml/templates/building.js

```diff
@@ -34,10 +34,10 @@ export const getBuilding =
     {{/if}}
     {{#if model.realestateIdentification.number}}
       <ns2:realestateIdentification>
-        <number>{{model.realestateIdentification.number}}</number>
         {{#if model.realestateIdentification.EGRID}}
           <EGRID>{{model.realestateIdentification.EGRID}}</EGRID>
         {{/if}}
+        {{{modelField model.realestateIdentification "number" namespace=""}}}
```

Comment:
Does the API accept the different order?

Reply:
Sadly Yep